States/State.py

```diff
@@ -329,6 +342,11 @@ def localSave(self):
         self.saveData["runs"].append({
             "startTime": self.staticStartTime.isoformat(),
```

Comment:
I don't think there's a point in having these anymore, we can just read the start of the first session and the end of the last session when we send this to `.lss`.

Reply:
Fixed.